x-pack/plugins/ml/server/models/data_recognizer/modules/security_linux/ml/v2_linux_rare_metadata_user.json

```diff
@@ -0,0 +1,35 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "This is a new refactored job which works on ECS compatible events across multiple indices. Security: Linux - Looks for anomalous access to the metadata service by an unusual user. The metadata service may be targeted in order to harvest credentials or user data scripts containing secrets.",
+  "groups": [
+    "security",
+    "auditbeat",
+    "endpoint",
+    "linux",
+    "process"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"user.name\"",
+        "function": "rare",
+        "by_field_name": "user.name"
+      }
+    ],
+    "influencers": [
+      "agent.id",
+      "host.name"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "32mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "created_by": "ml-module-security-linux"
```

Comment:
Is there a reason why we don't have the custom_urls to the security app for this job?

Reply:
Same reason as https://github.com/elastic/kibana/pull/71323 -  These fields, in events from cloud workloads, do not present super well in conventional network / host pages and the primary workflow will be in the detections and timeline pages after these get turned into detection alerts. A cloud username is more of an entity but we don't have a user page yet.